README.md

```diff
@@ -187,6 +187,36 @@ production as a docker image and push it to Docker hub at
 The image is then launched as a service in the next CI/CD job. `checklinks` is run against the
 running gatsby site in the docker container to check all links.
 
+## Process for using `devhub/latest` and `latest` tags on templates
+
+The devhub team uses the node template for tutorials and guides which requires making sure that both the templates and the write-ups (including solutions) are always up to date.
+We have a system that ensures that content in our documentation hub always builds correctly by using the `devhub/latest` tag for any Substrate crate. 
+
+This requires following a few steps for every new release of Substrate upstream:
+
+1. Go to the [release tags in the Substrate repository](https://github.com/paritytech/substrate/tags).
+1. Set the `devhub/latest` tag to the latest [Polkadot release](https://github.com/paritytech/polkadot/releases) (`polakdot-v<Major>.<Minor>.<Patch>`, not the release candidates apppended `...-rc#`).
```

Comment:
@NukeManDan : FYI we used Substrate's latest [here](https://github.com/paritytech/substrate/releases/tag/devhub%2Flatest).